bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/profiles/DefaultMasterProfile.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.internal.profiles;
+
+import java.util.concurrent.TimeoutException;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.common.SafeMethodCaller;
+import org.eclipse.smarthome.core.events.EventPublisher;
+import org.eclipse.smarthome.core.items.Item;
+import org.eclipse.smarthome.core.items.ItemUtil;
+import org.eclipse.smarthome.core.items.events.ItemEventFactory;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.thing.link.ItemChannelLink;
+import org.eclipse.smarthome.core.thing.profiles.StateProfile;
+import org.eclipse.smarthome.core.thing.util.ThingHandlerHelper;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is the default profile for stateful channels.
+ *
+ * It forwards commands to the {@link ThingHandler}. In the other direction it posts events to the event bus
+ * for state updates and commands which are initiated by the handler.
```

Comment:
"...and commands which are initiated by the handler" - was that really a use case for a "master" device? Do you have an example?

Reply:
Really good point - I just mindlessly took that from the old API. Now thinking about it, I don't have a good example. 

No binding in ESH is using it. 

In openhab2-addons there are only two bindings using it: [lutron](https://github.com/openhab/openhab2-addons/blob/50f7f662b4e5cf9f99b0c9a3e64f7097357d17e4/addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/SwitchHandler.java#L75) and [squeezebox](https://github.com/openhab/openhab2-addons/blob/52f768a3f8f80ac30d98dfb286bac134116baf99/addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxPlayerHandler.java#L395). I'm not familiar with both yet, so it's hard to judge whether this in fact is correct or not. On a first glance it looks like it might not be needed.

Reply:
PS: Just do double-check I read you correctly: This is not only about the JavaDoc, but you are suggesting to make master's "postCommand" a no-op. Correct?

Reply:
Correct.